### ISSUE TITLE
PyPI release

### ISSUE THREAD

**Issue by ocefpaf** [pyugrid/pyugrid#59]
Consider registering `pyugrid` at PyPI.  It would make make the life of `pip` users a little easier.

**Comment by ChrisBarker-NOAA** [comment]
Hmm -- do we think it is mature enough? If it's going to need looking at / messing around with the source code to be useful, then it's probably better for folks to get it from gitHub, rather than a pip install.

But maybe it is ready-enough?

-CHB

**Comment by ocefpaf** [comment]
I like the to follow the  the rule: Release early, release often.  (Adding a `Beta` release tag in the classifiers though :wink:)

**Comment by ChrisBarker-NOAA** [comment]
FAir enough -- done!

It hasn't shown up yet -- I don't know how long that takes.

At the moment, I think it will only have a link to gitHub.

Maybe we should put up an sdist?

**Comment by ocefpaf** [comment]
> Maybe we should put up an sdist?

:+1:

**Comment by ChrisBarker-NOAA** [comment]
OK -- done.

Please test it out in a bit....

**Comment by daf** [comment]
Seems to have installed:

```
~ -> pip install pyugrid                                                                                                                                                      
Downloading/unpacking pyugrid
  Downloading pyugrid-0.1.2.tar.gz
  Running setup.py (path:/tmp/pip_build_daf/pyugrid/setup.py) egg_info for package pyugrid
    
Requirement already satisfied (use --upgrade to upgrade): numpy in ./miniconda/envs/temp2/lib/python2.7/site-packages (from pyugrid)
Requirement already satisfied (use --upgrade to upgrade): netCDF4 in ./miniconda/envs/temp2/lib/python2.7/site-packages (from pyugrid)
Installing collected packages: pyugrid
  Running setup.py install for pyugrid
    
Successfully installed pyugrid
Cleaning up...
~ -> pip freeze | grep pyugrid                                                                                                                                                
pyugrid==0.1.2
```

**Comment by ocefpaf** [comment]
Ops... @daf was faster then me :stuck_out_tongue_winking_eye:, but everything worked here as well.  Thanks @ChrisBarker-NOAA.

**Comment by ChrisBarker-NOAA** [comment]
check this out:


   - 75 downloads in the last day

I'm guessing that's mirrors or something, but gratifying none the less :-)

-Chris


On Tue, Jun 9, 2015 at 10:05 AM, Filipe <notifications@github.com> wrote:

> Closed #59 <https://github.com/pyugrid/pyugrid/issues/59>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pyugrid/pyugrid/issues/59#event-326538383>.
>



-- 

Christopher Barker, Ph.D.
Oceanographer

Emergency Response Division
NOAA/NOS/OR&R            (206) 526-6959   voice
7600 Sand Point Way NE   (206) 526-6329   fax
Seattle, WA  98115       (206) 526-6317   main reception

Chris.Barker@noaa.gov

**Comment by ocefpaf** [comment]
@ChrisBarker-NOAA if you are into that kind if stats take a look at these:

https://binstar.org/IOOS/pyugrid/files
